corehq/apps/export/esaccessors.py

```diff
@@ -40,3 +40,9 @@ def get_ledger_section_entry_combinations(domain):
     ]
     query = LedgerES().domain(domain)
     return NestedTermAggregationsHelper(base_query=query, terms=terms).get_data()
+
+
+def get_case_name(case_id):
```

Comment:
another nit: I have an unwritten rule that anytime I look up a doc by its id, I also make sure it's in the domain I expect. Being strict about that makes you not have to rely on reasoning about the millions of ways (malicious or innocent) a case id might end up referenced in a form in the wrong domain, for example.

That said, I see this isn't something you added, so I'm happy to leave for another PR. But if it's easy, it'd be tighter if `_cached_case_id_to_case_name` required a `domain` argument and passed it in to this, and the search was limited by that domain.